exporter/datasetexporter/README.md

```diff
@@ -24,6 +24,24 @@ See the [Getting Started](https://app.scalyr.com/help/getting-started) guide.
 
 If you do not want to specify `api_key` in the file, you can use the [builtin functionality](https://opentelemetry.io/docs/collector/configuration/#configuration-environment-variables) and use `api_key: ${env:DATASET_API_KEY}`.
 
+### Server Host Settings
+
+Specifying the server host is crucial for ensuring the correct functionality of DataSet. 
```

Comment:
Thanks.

I think that's fine for now, but eventually it would be good to include some concrete examples - example input data (LogRecord), processor definition and how the output aka DataSet event will look like.

Reply:
EDIT: I just saw there is an example processor definition below.

I think it would be nice if in the future we can expand it with sample input data, processor and output - this way it's easier to visualuze it.